src/natcap/invest/sdr/sdr_core.pyx

```diff
@@ -374,7 +374,7 @@ def calculate_sediment_deposition(
 
     """
     LOGGER.info('Calculate sediment deposition')
-    cdef float sediment_deposition_nodata = -1.0
+    cdef float sediment_deposition_nodata = -1
```

Comment:
I think it's more readable to have the -1.0 since it is a float, was there a reason for trimming it?

Reply:
i believe trailing `.0`s are discouraged since they're usually left over from python 2 when `1.0` didn't behave the same as `1`. do you prefer it for cython specifically?

Reply:
Yea I was thinking about Cython specifically, since we define it as a float but then set it to a 'perceived' integer. But, I guess I'd defer to the latest Cython style / docs, if there are any.